common/templates/_images.tpl

```diff
@@ -0,0 +1,68 @@
+{{/*
+Return the proper image name
+{{ include "common.images.image" ( dict "imageRoot" .Values.path.to.the.image "global" .Values.global ) }}
+*/}}
+{{- define "common.images.image" -}}
+{{- $registryName := .imageRoot.registry -}}
+{{- $repositoryName := .imageRoot.repository -}}
+{{- $separator := ":" -}}
+{{- $termination := .imageRoot.tag | toString -}}
+{{- if .global }}
+    {{- if .global.imageRegistry }}
+     {{- $registryName = .global.imageRegistry -}}
```

Comment:
this would mean that a global registry being set would override all specific options for individual images, right? do we want to keep it this way or should this be the other way round?

Reply:
intuitively I thought this is what we wanted, apart from lifecycle + scheduler each chart has exactly one image, I can specify each image registry in the single image chart from their value ex metricsOperator.image.repository. So I tought the usecase was to override all... Do you think it would make more sense to have global as default and override if a specific one is written down?  So we will have to change our current charts to specify the global value for the release 🤔

Reply:
thank you for the explanation - i think in that case it makes sense to keep the current behavior